.github/workflows/ci.yml

```diff
@@ -286,6 +286,10 @@ jobs:
         run: py -3 test\util\rpcauth-test.py
 
       - name: Run functional tests
-        env:
-          TEST_RUNNER_EXTRA: ${{ github.event_name != 'pull_request' && '--extended' || '' }}
-        run: py -3 test\functional\test_runner.py --jobs $env:NUMBER_OF_PROCESSORS --ci --quiet --tmpdirprefix=$env:RUNNER_TEMP --combinedlogslen=99999999 --timeout-factor=$env:TEST_RUNNER_TIMEOUT_FACTOR $env:TEST_RUNNER_EXTRA
+        # TODO: Only run functional tests in master.
```

Comment:
I have removed the TODO. I can't say I'm a fan of TODOs in the code in general but since we use them it would also be fine to use them as "this should still be investigated" or "maybe check again if this has been fixed by now" reminders. I will open a tracking issue once this is merged.